zulip_bots/zulip_bots/game_handler.py

```diff
@@ -134,8 +134,9 @@ def usage(self) -> str:
     def initialize(self, bot_handler: Any) -> None:
         self.bot_handler = bot_handler
         self.get_user_cache()
-        self.email = self.bot_handler.email
-        self.full_name = self.bot_handler.full_name
+        self.config_info = bot_handler.get_config_info('tictactoe')
+        self.email = self.config_info.get("email", False)
+        self.full_name = self.config_info.get("full_name", False)
```

Comment:
The original code looks correct here and should remain unchanged. Rather, we probably need to update the TerminalBotHandler class in simple_lib.py (in specific, the __init__ method).